src/package.xml

```diff
@@ -1,363 +1,312 @@
-		<transformer name="TotalFuelConsumptionMap" displayName="Total Fuel Consumption Map by Iteration and Fire ID" className="SyncroSim.StochasticTime.Forms.StochasticTimeExportTransformer" classAssembly="SyncroSim.StochasticTime.Forms" dataSheet="OutputTotalFuelConsumptionMap" column="FileName" isFolderExport="True"/>
-	</transformers>
-	<layouts>
-		<layout name="coreforms_ResultTransformers" appendTo="burnP3Plus_Export">
-			<group name="Export" displayName="Prometheus">
-				<item name="RateOfSpreadMap" displayName="Rate of Spread Maps"/>
-				<item name="FireIntensityMap" displayName="Fire Intensity Maps"/>
-				<item name="SpreadDirectionMap" displayName="Spread Direction Maps"/>
-				<item name="SurfaceFuelConsumptionMap" displayName="Surface Fuel Consumption Maps"/>
-				<item name="CrownFractionBurnedMap" displayName="Crown Fraction Burned Maps"/>
-				<item name="CrownFractionConsumedMap" displayName="Crown Fraction Consumed Maps"/>
-				<item name="TotalFuelConsumptionMap" displayName="Total Fuel Consumption Maps"/>
-			</group>
-		</layout>
-		<layout name="corestimeforms_Maps">
-			<group name="PrometheusSecondaryOutputs" displayName="Prometheus Secondary Outputs">
-				<item name="ros" displayName="Rate of Spread" dataSheet="OutputRateOfSpreadMap" column="FileName"/>
-				<item name="fi" displayName="Fire Intensity" dataSheet="OutputFireIntensityMap" column="FileName"/>
-				<item name="sd" displayName="Spread Direction" dataSheet="OutputSpreadDirectionMap" column="FileName"/>
-				<item name="sfc" displayName="Surface Fuel Consumption" dataSheet="OutputSurfaceFuelConsumptionMap" column="FileName"/>
-				<item name="cfb" displayName="Crown Fraction Burned" dataSheet="OutputCrownFractionBurnedMap" column="FileName"/>
-				<item name="cfc" displayName="Crown Fraction Consumed" dataSheet="OutputCrownFractionConsumedMap" column="FileName"/>
-				<item name="tfc" displayName="Total Fuel Consumption" dataSheet="OutputTotalFuelConsumptionMap" column="FileName"/>
-			</group>
-		</layout>
-		<layout name ="coreforms_ProjectDatafeeds" appendTo="burnP3Plus_FuelDefinition">
-	  	<item name="FuelCodeCrosswalk" displayName="Prometheus Crosswalk"/>
-	  </layout>
-	  <layout name="coreforms_ScenarioDatafeeds" appendTo="burnP3Plus_OutputOption">
-			<item name="OutputOptionSpatial" displayName="Prometheus" showIf="inPipeline"/>
-	  </layout>
-	</layouts>
-	<updateProvider 
-		schemaVersion="2" 
-		minPackageVersion="0.4.11" 
-		className="SyncroSim.Core.XMLUpdateProvider" 
-		classAssembly="SyncroSim.Core"/>
-</package>
+<package name="burnP3PlusPrometheus" displayName= " BurnP3+ (Prometheus)" description= "Prometheus fire growth model for BurnP3+" version="2.0.0" url="https://github.com/burnp3/burnp3plusprometheus">
```

Comment:
Can you change displayName to "BurnP3+Prometheus"?

Reply:
Just remove the space from the beginning of the display name and it should be good!
" BurnP3+ (Prometheus)" --> "BurnP3+ (Prometheus)"